modules/spring-config/src/main/java/org/mule/runtime/config/spring/SpringRegistryLifecycleManager.java

```diff
@@ -106,7 +107,8 @@ public SpringContextInitialisePhase()
                     AbstractAsyncRequestReplyRequester.class,
                     OutboundRouter.class,
                     MessageProcessorChain.class,
-                    MuleContext.class
+                    MuleContext.class,
```

Comment:
May be possible to remove when changing the approach to provide the Service to the MuleContext